src/Firehose.Web/Authors/NigelSampson.cs

```diff
@@ -1,33 +0,0 @@
-﻿using System;
-using System.Collections.Generic;
-using System.Linq;
-using System.ServiceModel.Syndication;
-using Firehose.Web.Infrastructure;
-
-namespace Firehose.Web.Authors
-{
-	public class NigelSampson : IAmAMicrosoftMVP, IFilterMyBlogPosts
-	{
-		public string FirstName => "Nigel";
-		public string LastName => "Sampson";
-		public string StateOrRegion => "New Zealand";
-		public string EmailAddress => "nigel.sampson@compiledexperience.com";
-		public string ShortBioOrTagLine => "is the project lead of Caliburn.Micro and is often a developer at Pushpay";
-		public Uri WebSite => new Uri("https://compiledexperience.com/");
-		public string TwitterHandle => "nigelsampson";
-		public string GitHubHandle => "nigel-sampson";
-		public string GravatarHash => "ffccec4c43820d5370ad22de8d569d77";
-		public IEnumerable<Uri> FeedUris => new [] { new Uri("https://compiledexperience.com/blog/syndicate/atom/") };
```

Comment:
If we change the feed to https://compiledexperience.com/blog/syndicate/rss/ it should parse.
I suspect we may still want to exclude based on lack of xamarin content